### ISSUE TITLE
Cryptic Slide in WEB103:Hosting 

### ISSUE THREAD

**Issue by theitsystem** [gdisf/teaching-materials#299]
Slide 5 http://www.teaching-materials.org/hosting/slides.html#slide5

The bars do not clearly represent what shared resources on a server look like. Break that down into more chunks, colors, and express visually that each bar is dedicated to an app/site/company while expressing that the machine has limited cores, memory, and one connection to the web. It will make better sense that way.

#261

**Comment by gibbymt** [comment]
@theitsystem I agree that more useful pictures of the servers would better drive the point home. The images seem really out of place.

Also we should add Elastic Beanstalk to the PaaS
https://aws.amazon.com/elasticbeanstalk/
